packages/prover/package.json

```diff
@@ -15,8 +15,10 @@
   "type": "module",
   "exports": {
     ".": {
-      "import": "./lib/index.js",
-      "browser": "./lib/index.web.js"
+      "import": "./lib/index.js"
+    },
+    "./browser": {
+      "import": "./lib/browser/index.js"
```

Comment:
Had to make this chance, because conditional exports were not equally compatible with different libraries and tools. 

Named exports on the other hand make it more easy for all web builders and tools.